packages/mui-codemod/src/v5.0.0/base-remove-component-prop.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @param {import('jscodeshift').FileInfo} file
+ * @param {import('jscodeshift').API} api
+ */
+export default function transformer(file, api, options) {
+  const j = api.jscodeshift;
+  const root = j(file.source);
+  const printOptions = options.printOptions;
+
+  const transformed = root.findJSXElements().forEach((elementPath) => {
+    // Process only unstyled components
+    if (
+      !elementPath.node.openingElement.name.name.endsWith('Unstyled') ||
```

Comment:
This contradicts with the other codemod for removing this suffix, can be instead change the logic to look for imports from @mui/base?

Reply:
addressed